src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -401,52 +407,72 @@ export default defineComponent({
     /**
      * These layout-conditional props and listeners allow us
      * to set properties on the parent element depending on
-     * the layout in use. This is currently relevant for the
-     * boxed layout exclusively.
+     * the layout in use.
      */
-    const isBoxed = computed(() => props.layout === 'box')
-    const i18n = useI18n()
-    const layoutBasedProps = computed(() => {
-      if (!isBoxed.value) return {}
-      return {
-        href: `/audio/${props.audio.id}`,
-        class:
-          'block focus:bg-white focus:border-tx focus:ring-[3px] focus:ring-pink focus:ring-offset-[3px] focus:outline-none rounded-sm overflow-hidden cursor-pointer',
-      }
-    })
+    const isComposite = computed(() => ['box', 'row'].includes(props.layout))
+    const layoutBasedProps = computed(() =>
+      isComposite.value
+        ? {
+            href: `/audio/${props.audio.id}`,
+            class: [
+              'cursor-pointer',
+              {
+                'focus:ring-offset-[3px] focus:ring-[3px]':
+                  props.layout === 'box',
+                'focus:ring-offset-0 focus:ring-[1.5px]':
+                  props.layout === 'row',
+              },
+            ],
+          }
+        : {}
+    )
     const ariaLabel = computed(() =>
-      isBoxed.value
+      isComposite
```

Comment:
If we're not allowing go inside the composite audio component then we should add the author, license and category to the label, otherwise, it's not announced and screen reader users will not receive the same information.